src/ImageManipulation.php

```diff
@@ -1150,8 +1150,8 @@ protected function getDefaultAttributes(): array
         $attributes = [];
         if ($this->getIsImage()) {
             $attributes = [
-                'width' => $this->getWidth(),
-                'height' => $this->getHeight(),
+                'width' => $this->getWidth() ?: null,
```

Comment:
When the short code was rendering an non-existent image, `getWidth` and `getHeight` would return 0. This would lead to a `width="0"` attribute in the HTML.

Defaulting to null here fixes this.